sample_files/pre-commit-13.0/.isort.cfg

```diff
@@ -9,4 +9,3 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=
```

Comment:
Missing the default section, see https://github.com/OCA/maintainer-quality-tools/pull/629#issuecomment-588131706.